specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/azureFirewall.json

```
@@ -880,6 +884,34 @@
         "name": "AzureFirewallApplicationRuleProtocolType",
         "modelAsString": true
       }
+    },
+    "AzureFirewallSku": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of an Azure Firewall SKU.",
+          "enum": [
+            "AZFW_VNet",
+            "AZFW_Hub"  
+          ],
+          "x-ms-enum": {
+            "name": "AzureFirewallSkuName",
+            "modelAsString": true
+          }
+        },
+        "tier": {
+          "type": "string",
+          "description": "Tier of an Azure Firewall.",
```

```suggestion
          "description": "Tier of an Azure Firewall SKU.",
```
To keep consistency with SKU's name